content/installation/ElasticSearch/_index.md

```diff
@@ -20,17 +20,24 @@ toc = true
 - Deploy BotKube backend using **helm install** in your cluster:
 
   ```bash
-  $ helm install --version v0.12.4 botkube --namespace botkube --create-namespace \
-  --set communications.elasticsearch.enabled=true \
-  --set communications.elasticsearch.server=<ELASTICSEARCH_ADDRESS> \
-  --set communications.elasticsearch.username=<ELASTICSEARCH_USERNAME> \
-  --set communications.elasticsearch.password=<ELASTICSEARCH_PASSWORD> \
-  --set communications.elasticsearch.index.name=<ELASTICSEARCH_INDEX_NAME> \
-  --set communications.elasticsearch.index.type=<ELASTICSEARCH_INDEX_TYPE> \
-  --set communications.elasticsearch.index.shards=<ELASTICSEARCH_INDEX_SHARDS> \
-  --set communications.elasticsearch.index.replicas=<ELASTICSEARCH_INDEX_REPLICAS> \
-  --set config.settings.clustername=<CLUSTER_NAME> \
-  --set image.tag=v0.12.4 \
+  export ELASTICSEARCH_ADDRESS={elasticsearch_address}
+  export ELASTICSEARCH_USERNAME={elasticsearch_username}
+  export ELASTICSEARCH_PASSWORD={elasticsearch_password}
+  export ELASTICSEARCH_INDEX_NAME={elasticsearch_index_name}  
+  export ELASTICSEARCH_INDEX_SHARDS={elasticsearch_index_shards}  
+  export ELASTICSEARCH_INDEX_REPLICAS={elasticsearch_index_replicas}
+  export CLUSTER_NAME={cluster_name}  
+  
+  $ helm install --version v0.13.0 botkube --namespace botkube --create-namespace \
+  --set communications.default-group.elasticsearch.enabled=true \
+  --set communications.default-group.elasticsearch.server=<ELASTICSEARCH_ADDRESS> \
+  --set communications.default-group.elasticsearch.username=<ELASTICSEARCH_USERNAME> \
+  --set communications.default-group.elasticsearch.password=<ELASTICSEARCH_PASSWORD> \
+  --set communications.default-group.elasticsearch.indices.default.name=<ELASTICSEARCH_INDEX_NAME> \
+  --set communications.default-group.elasticsearch.indices.default.shards=<ELASTICSEARCH_INDEX_SHARDS> \
+  --set communications.default-group.elasticsearch.indices.default.replicas=<ELASTICSEARCH_INDEX_REPLICAS> \
+  --set settings.clusterName=<CLUSTER_NAME> \
+  --set image.tag=v0.13.0 \
```

Comment:
this can be removed, as this will be a default 👍

Reply:
of course.